test/verify/check-machines-create

```diff
@@ -122,11 +122,14 @@ class TestMachinesCreate(VirtualMachinesCase):
         runner = TestMachinesCreate.CreateVmRunner(self)
         config = TestMachinesCreate.TestCreateConfig
 
+        b = self.browser
+        m = self.machine
+
         self.login_and_go("/machines")
         self.browser.wait_in_text("body", "Virtual machines")
 
         # try to CREATE few machines
-        if self.machine.image in ['debian-stable', 'debian-testing', 'centos-8-stream']:
+        if m.image in ['debian-stable', 'debian-testing', 'centos-8-stream']:
```

Comment:
FTW, it seems really weird that this hasn't worked on centos-8-stream for so long. Are we missing some package there? It should be even al little ahead of 8.4

Reply:
The check is wrong, it will never be present this selector, we need to open the dialog before. I will post a fix separately, ok?

I am sure debian-testing has this option as well.

Reply:
(This PR does not touch affect this condition. Therefore I said follow up is fine)

Reply:
Yes, I know it's not new, it just made me raise an eyebrow as that is suspiciuos. Follow-up is totally fine of course.